ts/attribution-control.directive.ts

```diff
@@ -31,7 +31,7 @@ export class AttributionControlDirective extends Control.Attribution implements
     constructor(
         @Inject(forwardRef(() => MapComponent)) mapComponent: MapComponent
     ) {
-        super({prefix: '<a href="http://yagajs.org" title="A TypeScript library for interactive maps with Leaflet in Angular2">YAGA</a>'});
+        super({prefix: '<a href="https://yagajs.org" title="YAGA">YAGA</a> | <a href="https://leaflet-ng2.yagajs.org" title="A TypeScript library for interactive maps with Leaflet in Angular2">leaflet-ng2</a>'});
```

Comment:
This line exceeded the max-line-length of 140 chars. There are two ways:

1. Put the string content into a constant and place it on the top of the file or in consts.ts
2. Add an exception for linting inline before.

#### Example for Number 2:
```ts
/* tslint:disable:max-line-length */
super({prefix: '...'});
/* tslint:enable */
```